src/components/spatial/tiling-utils.js

```diff
@@ -27,10 +28,11 @@ function pixelsToTileIndex(pixelCount, z, tileSize) {
  * than minZoom, return an empty array. If the current zoom level is greater than maxZoom,
  * return tiles that are on maxZoom.
  */
-// This has been changed from deck.gl
-export function getTileIndices(viewport, maxZoom, minZoom, tileSize) {
+export function getTileIndices(viewport, maxZoom, minZoom, tileSize, width, height) {
```

Comment:
params list is getting long... Consider passing an object instead?

Reply:
I second the above.